.github/workflows/windows.yml

```
@@ -406,6 +421,13 @@ jobs:
           config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DCURL_USE_LIBPSL=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_WOLFSSL=ON -DCURL_USE_GSASL=ON'
       fail-fast: false
     steps:
+      - name: 'install stunnel'
+        if: ${{ matrix.tflags != 'skipall' && matrix.tflags != 'skiprun' }}
+        run: |
+          choco install -y stunnel --force
+          echo %PROGRAMFILES(X86)%\stunnel\bin>>%GITHUB_PATH%
+        shell: cmd
```

```suggestion
        run: choco install --yes --no-progress --limit-output --timeout 180 stunnel --force
```
Instead of modifying the global PATH, how about appending it in the test run step?

---

Updated